mypyc/ir/ops.py

```diff
@@ -1138,6 +1138,33 @@ def accept(self, visitor: 'OpVisitor[T]') -> T:
         return visitor.visit_raise_standard_error(self)
 
 
+class CFunctionCall(RegisterOp):
+    """ret = func_call(arg0, arg1, ...)
+
+    A call to a C function
+    """
+
+    error_kind = ERR_MAGIC
+
+    def __init__(self, function_name: str, args: List[Value], ret_type: RType, line: int) -> None:
+        super().__init__(line)
+        self.function_name = function_name
+        self.args = args
+        # TODO: handle void C function by adding a new flag and make this Optional
+        self.type = ret_type
+
+    def to_str(self, env: Environment) -> str:
+        args_str = ', '.join(env.format('%r', arg) for arg in self.args)
+        # TODO: comment: c_function_call to distinguish from cast, box, etc
+        return env.format('%r = %s(%s) :: c_function_call', self, self.function_name, args_str)
```

Comment:
I think that the `:: c_function_call` suffix is too verbose. We can leave it out for now, even though this results in some ambiguity. f/when we merge `Call` and this new class, we can figure out how to disambiguate different names.